endpointmanager/pkg/chplendpointquerier/allscriptsquerier.go

```diff
@@ -0,0 +1,74 @@
+package chplendpointquerier
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	http "net/http"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/chromedp/chromedp"
+)
+
+func AllScriptsQuerier(allscriptsURL string, fileToWriteTo string) {
+
+	var endpointEntryList EndpointList
+
+	var DSTU2URL string
+
+	ctx, cancel := chromedp.NewContext(context.Background())
+	defer cancel()
+
+	var htmlContent string
+
+	// Chromedp will wait for webpage to run javascript code to generate api search results before grapping HTML
+	err := chromedp.Run(ctx,
+		chromedp.Navigate(allscriptsURL),
+		chromedp.OuterHTML("html", &htmlContent, chromedp.ByQuery),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	doc, err := goquery.NewDocumentFromReader(strings.NewReader(htmlContent))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	doc.Find("a").Each(func(index int, linkhtml *goquery.Selection) {
```

Comment:
Can you write a few comments in this file to clarify some of the more complicated lines (36, 41, 47, and any others you think could be useful)? Like I assume this tries to find a link in the page, but because I'm not familiar with goquery